playground/mocks/data/idp/idx/error-400-bad-request.json

```diff
@@ -0,0 +1,18 @@
+{
+  "version": "1.0.0",
+  "stateHandle": "02Vmpjwzxw6j4wJEBjVzHMh_96dCSsUNW54QIDnCS-",
+  "expiresAt": "2023-06-23T07:41:47.000Z",
+  "intent": "LOGIN",
+  "messages": {
+      "type": "array",
+      "value": [
+          {
+              "message": "Bad request.",
+              "i18n": {
+                  "key": "errors.E0000020"
```

Comment:
@yannongli-okta The API response is actually `"key": "E0000020"` (no `errors.` prefix) which is a server-side bug. Please create and assign a bug ticket to the owner of this endpoint to fix.

Reply:
created https://oktainc.atlassian.net/browse/OKTA-623672